services/explorer-api/src/http-server/open-api-spec.ts

```diff
@@ -0,0 +1,31 @@
+import { openApiPaths } from "./routes/index.js";
+
+export const genereateOpenApiSpec = () => ({
+  "openapi": "3.1.0",
+  "info": {
+    "title": "Aztec Scan API",
+    "version": "0.2.0",
+    "description": "API for exploring Aztec Network. Please note that this is a work in progress and the API is subject to change.",
+  },
+  "servers": [
+    {
+      "url": "https://api.chicmoz.info/v1/{apiKey}",
+      "variables": {
+        "apiKey": {
+          "default": "d1e2083a-660c-4314-a6f2-1d42f4b944f4",
```

Comment:
Do we keep this api this public??

Reply:
Yes, it's public in the frontend anyway. The reason is that we haven't implemented `auth`-service yet. But to keep the correct auth-flow we just hardcode it.

Reply:
https://github.com/aztlan-labs/chicmoz/issues/61